modules/api/src/main/java/org/apache/fluo/api/data/Bytes.java

```diff
@@ -355,6 +354,36 @@ public static final Bytes of(String s, Charset c) {
   }
 
   /**
+   * Checks if this has the passed prefix
+   * 
+   * @param prefix
+   * @return true or false
+   */
+  public boolean startsWith(Bytes prefix) {
+    for (int i = 0; i < prefix.length; i++) {
+      if (this.byteAt(i) != prefix.byteAt(i))
```

Comment:
`mvn clean verify` fails with checkstyle. Fluo prefers using braces, even for one-line if blocks.